Drop-In/src/main/java/com/braintreepayments/api/SelectPaymentMethodFragment.java

```diff
@@ -0,0 +1,168 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.ViewSwitcher;
+
+import androidx.annotation.VisibleForTesting;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.LinearSnapHelper;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.braintreepayments.api.dropin.R;
+
+import java.util.List;
+
+public class SelectPaymentMethodFragment extends Fragment implements SupportedPaymentMethodSelectedListener, VaultedPaymentMethodSelectedListener {
+
+    private ViewSwitcher mLoadingViewSwitcher;
+    private TextView mSupportedPaymentMethodsHeader;
+
+    @VisibleForTesting
+    protected ListView mSupportedPaymentMethodListView;
+
+    private View mVaultedPaymentMethodsContainer;
+    private RecyclerView mVaultedPaymentMethodsView;
+    private Button mVaultManagerButton;
+
+    private DropInRequest dropInRequest;
+
+    @VisibleForTesting
+    DropInViewModel dropInViewModel;
+
+    public SelectPaymentMethodFragment() {}
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        Bundle args = getArguments();
+        if (args != null) {
+            dropInRequest = args.getParcelable("EXTRA_DROP_IN_REQUEST");
+        }
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.bt_fragment_select_payment_method, container, false);
+
+        mLoadingViewSwitcher = view.findViewById(R.id.bt_loading_view_switcher);
+        mSupportedPaymentMethodsHeader = view.findViewById(R.id.bt_supported_payment_methods_header);
+        mSupportedPaymentMethodListView = view.findViewById(R.id.bt_supported_payment_methods);
+        mVaultedPaymentMethodsContainer = view.findViewById(R.id.bt_vaulted_payment_methods_wrapper);
+        mVaultedPaymentMethodsView = view.findViewById(R.id.bt_vaulted_payment_methods);
+        mVaultManagerButton = view.findViewById(R.id.bt_vault_edit_button);
```

Comment:
Unrelated to this PR but in testing this out, I remembered that the settings toggle for enabling vault manager isn't working - can we add a TODO to investigate that?

Reply:
Def!